src/providers/info.rs

```diff
@@ -235,7 +232,7 @@ impl DefaultInfoProvider<'_> {
     // TODO: Check Eviction policies: https://redis.io/topics/lru-cache
     fn load_token_info(&mut self, token: &String) -> ApiResult<Option<TokenInfo>> {
         self.check_token_cache()?;
```

Comment:
In the check function we don't need a separate key. We can just use `exists` https://github.com/mitsuhiko/redis-rs/blob/master/src/commands.rs#L361